clojurescript.md

```diff
@@ -0,0 +1,44 @@
+# ClojureScript
+
+Official website: https://github.com/clojure/clojurescript
+
+## The answers
+
+1. What does the type system look like? **Dynamic**
+
+   Derived from Clojure but with a few differences. Supports immutable, persistent
+   collections.
+   [[1]](https://github.com/clojure/clojurescript/wiki/Differences-from-Clojure#data-structures)
+
+2. Is there a stable version? **Sort of**
+
+   1.8.40 is described as a "stable release" but it's nothing more than the latest
+   cut from master. The numbering aligns with the Clojure version.
+
+3. Is it under active development? **Yes**
+
+   The latest version was released in March 2016. The number of commits has
+   dropped in recent months.
+   [[2]](https://github.com/clojure/clojurescript/releases)
+   [[3]](https://github.com/clojure/clojurescript/graphs/contributors)
+
+4. What is its [bus factor](https://en.wikipedia.org/wiki/Bus_factor)? **Small**
+
+   Development relies pretty heavily on the work of
+   [David Nolen](https://github.com/swannodette) but it was originally created
+   by Rich Hickey and is supported by the Clojure/core team.
+   [[4]](https://github.com/clojure/clojurescript/graphs/contributors)
+   [[5]](http://clojure.com/blog/2011/07/22/introducing-clojurescript.html)
+
+5. Does it support Dead Code Elimination? **Yes**
+
+   Through Google's Closure library.
```

Comment:
macro can do a great work in detect Dead Code Elimination. This is an example that actually work in clojure but don't in clojurescript.  
http://blog.venanti.us/yagni/